phys/module_sf_bem.F

```diff
@@ -1062,7 +1317,7 @@ subroutine int_rsrad(albwin,albwal,pwin,rswal,&
             enddo
 
 !We suppose that the radiation is spread isotropically within the
-!room when it passes through the windows, so the flux [W/m^2] in every 
+!room when it passes through the windows, so the flux [W/m\B2] in every 
```

Comment:
m^2 replaced - is ^ not allowed in comments?

Reply:
Jimy,
I agree. There is nothing wrong with the "^" character. In addition to this instance, there are two other locations that I am going to swap back to "W/m^2".

Reply:
Actually, there are MANY locations where "W/m2" is used in this file, so that is what I will use for the convention in this file.